MAUI/Cartesian-Charts/Get-the-data-point-collection-based-on-region.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Get the data point collection in Syncfusion SfCartesianChart
+description: Get the data points that fall inside a given rectangular area or within specified X and Y coordinate ranges in SfCartesianChart
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Get the data point collection based on region
+
+[CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html) provides the following methods to get a collection of data under a particular region.
+
+* [`GetDataPoints(Rect rectangle)`]() - Method to get the collection of data that fall inside the given rectangle region. 
+* [`GetDataPoints(double startX, double endX, double startY, double endY)`]() - Method to get the data points from the given range.
```

Comment:
Gets the collection of data from the given axis visible range.

Reply:
Modified the content